.circleci/config.yml

```diff
@@ -6,15 +6,19 @@ common-steps:
       paths:
         - "*"
 
-  - &removevirtualenv
-    run:
-      name: Removes the upstream virtualenv from the original container image
-      command: sudo pip uninstall virtualenv -y
-
   - &installdeps
     run:
       name: Install Debian packaging dependencies
-      command: make install-deps
+      command: |
+        apt-get update && apt-get install -y make sudo
+        make install-deps
+
+  - &installtestdeps
+    run:
+      name: Install test dependencies
+      command: |
+        sudo apt-get install reprotest faketime -y --no-install-recommends
+        .venv/bin/pip install -r test-requirements.txt
```

Comment:
Is using `.venv/bin/pip` equivalent to activating the virtual environment?

Reply:
It [should be](https://docs.python.org/3/library/venv.html) for `venv`:

> You don’t specifically need to activate an environment; activation just prepends the virtual environment’s binary directory to your path, so that “python” invokes the virtual environment’s Python interpreter and you can run installed scripts without having to use their full path. However, all scripts installed in a virtual environment should be runnable without activating it, and run with the virtual environment’s Python automatically.

Reply:
Yep. Note that there's a subtle gotcha I ran into in the reproducibility tests, which end up shelling out to plain `python`. If you don't explicitly activate the venv, that plain `python` will point to the system Python instead of our venv's. In theory you can work around it by using sys.executable() but I didn't want to go down that route in this PR.